tools/webpack.config.ts

```diff
@@ -147,8 +145,6 @@ export default (env?: string): webpack.Configuration[] => {
         { path: "../static/generated/emoji/emoji_codes.js" },
         { path: "../static/generated/pygments_data.js" },
         { path: "../static/js/debug.js" },
-        { path: "../static/js/blueslip.js" },
-        { path: "../static/js/common.js" },
```

Comment:
Why do we get to remove these?

Reply:
They already add themselves to `window`, so we don’t need to do it for them. And we don’t want to give them an expose-loader configuration because it would override the Babel configuration (at least the way we do it).

Reply:
Okay “override” might be too strong a word—both loaders were used, but they were interfering somehow with the result that the polyfills required by `common.js` (e.g. `core-js/modules/es.map`) were getting lost. An alternate solution is to move expose-loader to the beginning of the rules list, so I’ll just do that instead and come back for expose-loader with a shotgun later.